### PR TITLE
DDF-1191: Replaced -XX:-UseSplitVerifier with -noverify

### PR DESCRIPTION
**Pull request by AzGoalie**
@shaundmorris 
@lessarderic

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf-parent/24)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -380,7 +380,8 @@
                     <version>2.8.1</version>
                     <configuration>
                         <!-- -XX:-UseSplitVerifier is needed to make tests using powermock compile under java 7 -->
```
Comment (rzwiefel) [inline pom.xml]: @AzGoalie Shouldn't we remove this line now that the UseSplitVerifier flag isn't in there anymore?

File: pom.xml [hunk 2]
```diff
@@ -379,8 +379,8 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>2.8.1</version>
                     <configuration>
-                        <!-- -XX:-UseSplitVerifier is needed to make tests using powermock compile under java 7 -->
-                        <argLine>${argLine} -Djava.awt.headless=true -XX:-UseSplitVerifier</argLine>
+                        <!-- Replaced -XX:-UseSplitVerifier with -noverify for java 8 compatibility-->
```
Comment (lessarderic) [inline pom.xml]: We don't need the comment here. Already included in the commit message.

**Comment by shaundmorris** [comment]
:+1: 

---

Review status: 0 of 1 files reviewed, 2 unresolved discussions, some commit checks pending.
Reviewed files:
- pom.xml @ r2

---

<sup>**[pom.xml, line 382 [r2]](https://reviewable.io:443/reviews/codice/ddf-parent/24#-JrZJ_aiDAe_fHRkU2XG-r2-382)** ([raw file](https://github.com/codice/ddf-parent/blob/3372cf5bb048a3f46f510acf0990879f73698e9f/pom.xml#L382)):</sup>
I'm okay with that as well.

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf-parent/24)

**Comment by shaundmorris** [comment]
Verified build was good (jdk1.7 as well) plan on merging tomorrow

**Pull request merged**